Custom.pm

```diff
@@ -51,12 +53,16 @@ sub dirsFor {
 	
 	if ($dir eq 'updates') {
 
-        mkdir UPDATE_DIR unless -d UPDATE_DIR;
-        	
+		mkdir UPDATE_DIR unless -d UPDATE_DIR;
+
 		@dirs = (UPDATE_DIR);
 	}
 	else {
 		@dirs = $class->SUPER::dirsFor($dir);
+
+		if ($dir eq "Plugins") {
+			push @dirs, catdir( Slim::Utils::Prefs::preferences('server')->get('cachedir'), 'Plugins' );
+		}
```

Comment:
I took a look at this today.  He was neglecting to give us the real error.

`
[17-08-05 11:01:32.4769] Slim::bootstrap::tryModuleLoad (286) Warning: Module [Plugins::DenonAvpControl::Plugin] failed to load:
Can't locate Plugins/DenonAvpControl/Plugin.pm in @INC (you may need to install the Plugins::DenonAvpControl::Plugin module) (@INC contains: /usr/local/slimserver/Cache/InstalledPlugins /usr/local/slimserver/CPAN/arch/5.24/arm-linux-gnueabihf-thread-multi-64int /usr/local/slimserver/CPAN/arch/5.24/arm-linux-gnueabihf-thread-multi-64int/auto /usr/local/slimserver/CPAN/arch/5.24.0/arm-linux-gnueabihf-thread-multi-64int /usr/local/slimserver/CPAN/arch/5.24.0/arm-linux-gnueabihf-thread-multi-64int/auto /usr/local/slimserver/CPAN/arch/5.24/arm-linux-gnueabihf-thread-multi-64int /usr/local/slimserver/CPAN/arch/5.24/arm-linux-gnueabihf-thread-multi-64int/auto /usr/local/slimserver/CPAN/arch/arm-linux-gnueabihf-thread-multi-64int /usr/local/slimserver/CPAN/arch/5.24 /usr/local/slimserver/lib /usr/local/slimserver/CPAN /usr/local/slimserver /usr/local/slimserver /usr/local/lib/perl5/site_perl/5.24.0/arm-linux-gnueabihf-thread-multi-64int /usr/local/lib/perl5/site_perl/5.24.0 /usr/local/lib/perl5/5.24.0/arm-linux-gnueabihf-thread-multi-64int /usr/local/lib/perl5/5.24.0 /usr/local/lib/perl5/site_perl . ) at (eval 892) line 1.
BEGIN failed--compilation aborted at (eval 892) line 1.
`

Is this the right way to fix it?

               if ($dir eq "Plugins") {
                        push @dirs, catdir( Slim::Utils::Prefs::preferences('server')->get('cachedir'), 'Plugins' );
                        push @INC, catdir( Slim::Utils::Prefs::preferences('server')->get('cachedir') );
                }